cloudflare/src/endpoints/workers/create_secret.rs

```diff
@@ -0,0 +1,37 @@
+use super::WorkersSecret;
+
+use crate::framework::endpoint::{Endpoint, Method};
+
+/// Create Secret
+/// https://api.cloudflare.com/#worker-create-secret
+pub struct CreateSecret<'a> {
+    /// account id of owner of the script
+    pub account_identifier: &'a str,
+    /// the name of the script to attach the secret to
+    pub script_name: &'a str,
```

Comment:
I'm starting to think that these should be owned strings

Reply:
What do you mean by these? @ashleymichal how should we apply this suggestion?

Reply:
Basically turn `&'a str` into a `String`

Reply:
we don't do this anywhere else :-\

Reply:
@sssilver i actually started turning all of these `&'a str`'s into `&'static str`'s. saves the requirement for lifetimes on all your structs and their methods. will illustrate in another pr.